DakeAndDevileCorps/Screens/Store/VC/StoreDetailViewController.swift

```diff
@@ -118,7 +126,11 @@ extension StoreDetailViewController: UITableViewDataSource {
         case (.itemInformation, .productList):
             return productList.count
         case (.itemInformation, .reviewList):
-            return reviewList.count
+            if reviewList.isEmpty {
+                return 1
+            } else {
+                return reviewList.count
+            }
```

Comment:
p5;

```suggestion
            return reviewList.isEmpty ? 1 : reviewList.count
```
이렇게 하면 조금 짧아지지 않을까 싶어서 적어봅니다,

Reply:
[054e314](https://github.com/DeveloperAcademy-POSTECH/MC3-Team4-DakeAndDevileCorps/pull/87/commits/054e314768c7f317bb2262da25e0c5c8d668a15b) 반영했습니다!